src/js/worklets/onboarding_carousel.js

```diff
@@ -0,0 +1,60 @@
+import { useDerivedValue, withTiming, Easing } from 'react-native-reanimated';
+
+const slideAnimationDuration = 300;
+
+const easeOut = {
+  duration: slideAnimationDuration,
+  easing: Easing.bezier(0, 0, 0.58, 1),
+}
+
+// Derived Values
+export function dynamicProgressBarWidth (staticProgressBarWidth, progress) {
+  return useDerivedValue(
+    function () {
+      'worklet'
+      var progressValue = progress.value;
+      // Without this check, ios debug build is crashing
+      progressValue = isNaN(progressValue) ? 0 : progressValue;
```

Comment:
It will be awesome if someone with ios/mac setup can check if ios debug is working. No specific step, the app was just crashing on opening (login or intro screen)

Reply:
Will check

Reply:
Got an error when Enabling Debug Mode but not sure if it is related with this. Normal execution with `make run-metro` worked fine though

Reply:
Thank you for confirming, probably not related.
We can try the same thing in the develop to be sure. Or we can fix it separately if it causes the error.